sd_jwt/bin/sd_jwt

```diff
@@ -0,0 +1,303 @@
+#!/usr/bin/env python3
+import argparse
+import json
+import logging
+import sys
+
+from sd_jwt import __version__
+from sd_jwt.demo_utils import (
+    print_decoded_repr,
+    get_jwk, 
+    print_repr
+)
+from sd_jwt.operations import (
+    create_sd_jwt_and_svc,
+    create_release_jwt,
+    generate_salt,
+    verify
+)
+
+logger = logging.getLogger("sd_jwt")
+
+
+parser = argparse.ArgumentParser(
+    description=f"{__file__} demo.",
+    epilog=f"{__file__}",
+    formatter_class=argparse.RawTextHelpFormatter
+)
+parser.add_argument(
+    '-t', '--type', required=False,
+    choices=(
+        'simple',
+        'structured',
+        'complex',
+    ),
+    default='structured',
+    help="Debug level, see python logging; defaults to INFO if omitted"
+)
```

Comment:
The help text does not match the switch that is defined here.

Reply:
My bad, consider it done

Reply:
Done here
https://github.com/oauthstuff/draft-selective-disclosure-jwt/commit/cbf13bef73911da2f37b969d0cd7b7d1993d667b

Reply:
Ok, I'm changing that parameter with --example
completely agree, I also had some concerns about that, so your proposal is the solution

Reply:
Done here
[8c2643c](https://github.com/oauthstuff/draft-selective-disclosure-jwt/pull/61/commits/8c2643c6d51a095f28b07f955075205f79d550c4)